typed-racket-lib/typed-racket/typecheck/provide-handling.rkt

```diff
@@ -37,17 +37,18 @@
 ;; provides.
 ;;
 ;; The second value is a list of two element lists, which are type name aliases.
-(define (generate-prov defs provs pos-blame-id)
+(define (generate-prov defs provs pos-blame-id mk-redirect-id)
   ;; maps ids defined in this module to an identifier which is the possibly-contracted version of the key
   (define mapping (make-free-id-table))
 
-  ;; triple/c in the signatures corresponds to three values:
-  ;; (values syntax? identfier? (listof (list/c identifier? identifier?))
-  ;; First return value is a syntax object of definitions
-  ;; Second is the id to export
-  ;; Third is a list of two element lists representing type aliases
+  ;; quad/c in the signatures corresponds to four values:
```

Comment:
Just want to make sure I understand this change. So we need to split up the definition vs. just the id to export so that the definitions can be relocated into a submodule right?